chatbot/package.json

```diff
@@ -13,7 +13,7 @@
     "dotenv": "^6.1.0",
     "express": "^4.16.4",
     "express-session": "^1.15.6",
-    "lodash": "^4.17.15",
```

Comment:
I'm kind of surprised that it bothered to do this upgrade. Because we're using the `^` in our version number, it should already be getting the latest 4.x.x version (https://github.com/npm/node-semver#caret-ranges-123-025-004).